### PR TITLE
Use ta-scripts

### PR DESCRIPTION
**Pull request by levithomason**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -64,13 +64,16 @@ For a listing of development commands use the following:
...
+### Releasing
 
-    npm version <version>     # bump the new version
-    git push                  # push bump commit
-    git push --tags           # push version tag
+On the latest clean `master`:
+
+    npm run release:major
+    npm run release:minor
+    npm run release:patch
```
Comment (levithomason) [inline README.md:73]: Releases are now easy and explicit.  Thanks to our ta-scripts package, they are also consistent across our projects.

File: circle.yml [hunk 2]
```diff
@@ -9,7 +9,7 @@ machine:
 
 dependencies:
   pre:
-    - pip install awscli
+    - npm install -g npm@3
```
Comment (levithomason) [inline circle.yml:13]: `awscli` is used to sync to s3 (future dist assets).  The ta-script package will handle installing it if it doesn't exist.

File: scripts/circle_deploy.sh [hunk 3]
```diff
@@ -14,49 +13,14 @@ git config --global push.default simple
...
-
-if ${NPM_IS_PUBLISHED}; then
- echo "...$NPM_PACKAGE_NAME@$NPM_PACKAGE_VERSION is already published"
-else
- echo "...publishing $NPM_PACKAGE_NAME@$NPM_PACKAGE_VERSION"
- echo "...writing ~/.npmrc"
- echo "//registry.npmjs.org/:_authToken=${NPM_TOKEN}" > ~/.npmrc
- echo "...publishing"
- npm publish
-fi
```
Comment (levithomason) [inline scripts/circle_deploy.sh:59]: Bye bye publish garbage

File: scripts/circle_deploy.sh [hunk 4]
```diff
@@ -14,49 +13,14 @@ git config --global push.default simple
...
-git add CHANGELOG.md
-
-if [[ -n $(git status --porcelain) ]]; then
-  echo "...starting push, CHANGELOG.md is dirty after build"
-  git commit -n -m "deploy CHANGELOG.md by $CIRCLE_USERNAME [ci skip]"
-  git push origin $CIRCLE_BRANCH
-else
-  echo "...skipping push, repo is clean after build"
-fi
+ta-script circle_ci/create_changelog.sh
```
Comment (levithomason) [inline scripts/circle_deploy.sh]: All this logic is now handled by ta-scripts.  Also, it has been updated to a more proper approach.

**Comment by ghost** [comment]
Less is more :credit_card:

**Comment by dvdzkwsk** [comment]
:ghost:

File: circle.yml [hunk 5]
```diff
@@ -9,7 +9,8 @@ machine:
 
 dependencies:
   pre:
-    - pip install awscli
+    - echo "//registry.npmjs.org/:_authToken=${NPM_TOKEN}" > ~/.npmrc
+    - npm install -g npm@3
```
Comment (levithomason) [inline circle.yml:13]: awscli is used to sync to s3 (future dist assets). The ta-script package will handle installing it if it doesn't exist.

File: scripts/circle_deploy.sh [hunk 6]
```diff
@@ -14,49 +13,14 @@ git config --global push.default simple
...
-git add CHANGELOG.md
-
-if [[ -n $(git status --porcelain) ]]; then
-  echo "...starting push, CHANGELOG.md is dirty after build"
-  git commit -n -m "deploy CHANGELOG.md by $CIRCLE_USERNAME [ci skip]"
-  git push origin $CIRCLE_BRANCH
-else
-  echo "...skipping push, repo is clean after build"
-fi
+ta-script circle_ci/create_changelog
```
Comment (levithomason) [inline scripts/circle_deploy.sh:16]: Simple changelogs!

**Pull request merged**